### PR TITLE
Message functional tests - happy path complete

### PR DESCRIPTION
**Pull request by wang2208**
This PR also includes placeholders for a few unit test cases in MessageController and AccountController

### CODE REVIEW

File: twtr/grails-app/controllers/twtr/MessageController.groovy [hunk 1]
```diff
@@ -69,7 +69,7 @@ class MessageController extends RestfulController<Message> {
     def search(){
         def searchText = params.text
 
-        def results = Message.where {messageText ==~ "%${searchText}%"}.list()
+        def results = Message.where {messageText ==~ "%$searchText%"}.list()
```
Comment (donaldsawyer) [inline twtr/grails-app/controllers/twtr/MessageController.groovy:73]: if there's time, we should add the offsets & max to this.

File: twtr/src/integration-test/groovy/twtr/MessageFunctionalSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+
+    def 'return the most recent messages from an account with #goodId1'() {
+        given: 'create 12 more messages for #goodId1 to make 15 messages total'
+        def response
+        for (def i = 4; i <= 15; i++) {
+            response = restClient.post(path: '/message/tweet', query: [accountId: goodId1],
+                    contentType: 'application/json', body: [messageText: messageTemplate + i])
+            goodDateCreated[i - 1] = response.data.dateCreated
+        }
```
Comment (donaldsawyer) [inline twtr/src/integration-test/groovy/twtr/MessageFunctionalSpec.groovy:231]: (4..15).each { it -> ...code here... } is the 'groovy way' to do this :)

Comment (wang2208) [inline twtr/src/integration-test/groovy/twtr/MessageFunctionalSpec.groovy:231]: I realized that after I submitted the PR, will change it with my next commit.

Comment (donaldsawyer) [inline twtr/src/integration-test/groovy/twtr/MessageFunctionalSpec.groovy:231]: you can always do another commit and it will just add it to the PR that's open automatically.  But, I'm gonna merge this one.
